webapi/src/test/scala/org/knora/webapi/it/v2/KnoraSipiIntegrationV2ITSpec.scala

```diff
@@ -741,6 +698,35 @@ class KnoraSipiIntegrationV2ITSpec
       checkResponseOK(sipiGetFileRequest)
     }
 
+    "not create a document resource if the file is actually a zip file" in {
+      // Upload the file to Sipi.
+      val sipiUploadResponse: SipiUploadResponse = uploadToSipi(
+        loginToken = loginToken,
+        filesToUpload = Seq(FileToUpload(path = pathToMinimalZip, mimeType = MediaTypes.`application/zip`))
+      )
+
+      val uploadedFile: SipiUploadResponseEntry = sipiUploadResponse.uploadedFiles.head
+      uploadedFile.originalFilename should ===(minimalZipOriginalFilename)
+      uploadedFile.fileType should equal("bundle")
+
+      // Ask Knora to create the resource.
```

Comment:
```suggestion
      // Create the resource.
```

Reply:
Actually, this "ask Knora" comment is everywhere but is wrong/outdated (who the f... is Knora? ;). Maybe you could delete this in the whole file at once?

Reply:
done. (but there are still sooooo many places that talk of knora...)